src/tests/golden/quickfix_test.go

```diff
@@ -0,0 +1,123 @@
+package golden
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/VKCOM/noverify/src/linter"
+	"github.com/VKCOM/noverify/src/linttest"
+)
+
+func TestQuickFix(t *testing.T) {
+	s := newQuickFixTest(t, `testdata/quickfix`)
+	s.runQuickFixTest()
+}
+
+const (
+	expectedExtension = ".fix.expected"
+	fixExtension      = ".fix"
+)
+
+type quickFixTest struct {
+	t      *testing.T
+	folder string
+}
+
+func newQuickFixTest(t *testing.T, folder string) quickFixTest {
+	return quickFixTest{
+		t:      t,
+		folder: folder,
+	}
+}
+
+func openFile(filename string) (f *os.File, found bool, err error) {
+	f, err = os.Open(filename)
+	if err != nil {
+		f, err = os.Create(filename)
+		if err != nil {
+			return nil, false, fmt.Errorf("file creation %s failed: %s", filename, err)
+		}
+		return f, false, nil
+	}
+	return f, true, nil
+}
+
+func (t *quickFixTest) runQuickFixTest() {
+	files, err := linttest.FindPHPFiles(t.folder)
+	if err != nil {
+		t.t.Errorf("Error while searching for files in the %s folder: %s", t.folder, err)
```

Comment:
I would thing that this needs to be a `t.Fatalf` or we need to do a return after Errorf.
Should this test continue if we failed to get a files list?